src/main/java/uk/gov/pay/directdebit/payments/params/PaymentViewSearchParams.java

```diff
@@ -57,10 +77,19 @@ public PaginationParams getPaginationParams() {
     public String generateQuery() {
         StringBuilder sb = new StringBuilder("");
         if (searchDateParams.getFromDate() != null) {
-            sb.append("AND pr.created_date > :" + FROM_DATE_FIELD + " ");
+            sb.append(" AND pr.created_date > :" + FROM_DATE_FIELD);
         }
         if (searchDateParams.getToDate() != null) {
-            sb.append("AND pr.created_date < :" + TO_DATE_FIELD + " ");
+            sb.append(" AND pr.created_date < :" + TO_DATE_FIELD);
+        }
+        if (isNotBlank(email)) {
+            sb.append(" AND pa.email ILIKE :" + EMAIL_FIELD);
```

Comment:
Do we need to use `ILIKE` instead of `LIKE`? If we can skip `PostgreSQL` specific syntax we can be less vendor locked

Reply:
It's the same in Connector. It's a small compromise. I mean what are the chances that we suddenly swap DBS?

Reply:
Good point @SandorArpa ! I was not aware of performance optimisation, thanks!